templates/openstack/install.html

```diff
@@ -254,175 +276,757 @@ <h3>Installation instructions</h3>
+              <img src="https://assets.ubuntu.com/v1/ae8bcd36-maas+import+ssh+key.png?w=848"
+                width="424" height="241" alt="" />
+            </a>
+          </p>
+          <p>Then navigate to the &lsquo;Subnets&rdquo; tab in the top menu and press on the &ldquo;untagged&rdquo;
+          VLAN next to the &ldquo;172.16.7.0/24&rdquo; subnet (the <code>provisioning</code> subnet):</p>
+          <p>
+            <a href="https://assets.ubuntu.com/v1/cb64996c-maas+subnets.png">
+              <img src="https://assets.ubuntu.com/v1/cb64996c-maas+subnets.png?w=848"
+                width="424" height="241" alt="" />
+            </a>
+          </p>
+          <p>From the &ldquo;Take action&rdquo; drop-down menu on the right select &ldquo;Provide DHCP&rdquo;:</p>
+          <p>
+            <a href="https://assets.ubuntu.com/v1/baa98c38-maas+default+vlan.png">
+              <img src="https://assets.ubuntu.com/v1/baa98c38-maas+default+vlan.png?w=848"
+                width="424" height="241" alt="" />
+            </a>
+          </p>
+          <p>Fill in the &ldquo;Dynamic range start IP&rdquo; and &ldquo;Dynamic range end IP&rdquo; fields and
+          press the &ldquo;Provide DHCP&rdquo; button:</p>
           <p>
-            <a href="https://assets.ubuntu.com/v1/b4a8693c-build-openstack-with-conjure-up-1.png">
-              <img src="https://assets.ubuntu.com/v1/b4a8693c-build-openstack-with-conjure-up-1.png?w=424" width="424" alt="conjure-up's opening wizard in command line" />
+            <a href="https://assets.ubuntu.com/v1/d442bf3d-maas+dynamic+range+ip.png">
+              <img src="https://assets.ubuntu.com/v1/d442bf3d-maas+dynamic+range+ip.png?w=848"
+                width="424" height="241" alt="" />
             </a>
           </p>
+          <p>At this point, your MAAS instance is ready to provision other machines.</p>
         </div>
       </li>
 
       <li class="p-stepped-list__item">
-        <h4 class="p-stepped-list__title">Select OpenStack hypervisor</h4>
+        <h4 class="p-stepped-list__title">Bootstrap and configure other nodes</h4>
         <div class="p-stepped-list__content">
-          <p>conjure-up offers two options with OpenStack:</p>
-          <p>1. OpenStack with NovaLXD</p>
-          <p>This installs OpenStack configured to use the LXC &lsquo;machine container&rsquo; hypervisor &mdash; containers that behave like ultra-fast VMs. Choose this option to run Linux workloads in LXD containers, for higher density and for evaluation of OpenStack.</p>
-          <p>2. OpenStack with NovaKVM</p>
-          <p>This installs OpenStack configured to use the KVM hypervisor ready to launch full virtual machines. We recommend this option for production clouds.</p>
+          <p>All other nodes should be configured to PXE boot from the provisioning subnet
+          by default.</p>
+          <p>Once configured, power them on and wait until they show up in the ‘Machines’
```

Comment:
Double quotes are used for label names throughout the rest of these instructions:

```suggestion
          <p>Once configured, power them on and wait until they show up in the &ldquo;Machines&ldquo;
```